### DISCUSSION TITLE
Has anyone else found they no longer have to reboot for touch to work since 2024.6.0 ????

### DISCUSSION THREAD

**Discussion by BigBobbas** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#23] [Q&A]
if others can confirm this please :), then I can update the guides etc. But since updating, i no longer have to reboot the device after flashing or first powering up the device.

**Comment by Bluhme1** [comment]
No need to reboot at my end. Everything working fine - also touch

**Reply by BigBobbas** [reply]
perfect! thanks for the feedback . I can update the docs 👍

**Reply by youkorr** [reply]
everything is fine on my side also no need to reboot

**Comment by BigBobbas** [comment]
updates docs to reflect that this is no longer required

**Comment by Bcukier** [comment]
My touchscreen fails after every update.  A reset solves that.  After the boot, here is the relevant part of what appears if helpful:

`[06:38:08][C][gt911.touchscreen:110]: GT911 Touchscreen:
[06:38:08][C][gt911.touchscreen:111]:   Address: 0x5D
[06:38:08][C][gt911.touchscreen:112]:   Interrupt Pin: GPIO3
[06:38:08][E][component:082]:   Component touchscreen is marked FAILED `

**Reply by BigBobbas** [reply]
I'm looking into this still, there is no obvious reason as to why this is happening for some people and not for others. as soon as we find a solution we will post an update 🤞
